src/GridView/GridView.js

```diff
@@ -11,6 +11,19 @@ import config from './../config/config.js';
 import Header from "../Header";
 
 class GridView extends ConnectionComponent {
+	constructor() {
+		super();
+		this.getChildEntityTypes = this.getChildEntityTypes.bind(this);
+        this.getEntityType = this.getEntityType.bind(this);
```

Comment:
Please check you indentation settings, it looks fine in webstorm, but this line is indented with whitespace instead of tabs :trollface: